### PR TITLE
fixed broken suspend script / remove --onlyblocking option (fixes #4717)

### PR DESCRIPTION
**Pull request by chdoc**
Fixes: DFHack/dfhack#4717

### CODE REVIEW

**Review by myk002 — APPROVED** [review]

**Comment by chdoc** [comment]
Getting changes approved before the CI has run, that's a first for me :smiley:

**Comment by myk002** [comment]
> Getting changes approved before the CI has run, that's a first for me 😃

I still wait for CI to pass before merging tho : p

**Pull request merged**
